syncd/CommandLineOptions.h

```diff
@@ -27,6 +28,11 @@ namespace syncd
          * boot mode
          */
         SAI_START_TYPE_FASTFAST_BOOT = 3,
+	/**
```

Comment:
add empty line before comment, also was this new type boot discussed on SAI community meeting ? whether this is differetn from fast boot and fastfast boot ?

Reply:
can we have description what those modes actually do ?

Reply:
Added empty line. Express boot is a cisco proprietary implementation. It was presented in one of the meetings. It is different from fast boot. Fastfast boot is another vendor's implementation. I dont know much of it.